open_dread_rando/files/schema.json

```diff
@@ -407,10 +407,6 @@
             "additionalProperties": false,
             "default": {}
         },
-        "show_shields_on_minimap": {
```

Comment:
Let's keep this value in the schema to not break API, even if it does nothing. Though adding a description explaining it does nothing is helpful. Also the perfect use of calling it _deprecated_.